### ISSUE TITLE
Use robot account to create issues through contribution tool

### ISSUE THREAD

**Issue by MattiSG** [ambanum/OpenTermsArchive#313]
Issues created automatically through the contribution tool (e.g. #310, #311, #312…) appear as created by @martinratinaud. This makes it hard to follow through conversations and prioritise handling actual interventions by a human contributor.

The API token used to create these issues could be obtained from @AmbNum-Bot instead to solve this problem 🙂

**Comment by martinratinaud** [comment]
I totally agree and am waiting for Github's answer as I can't connect to the existing bot because of 2FA.
